### ISSUE TITLE
Rethemes and Redesigns. 

### ISSUE THREAD

**Issue by FreneticScribbler** [BrassGoggledCoders/SteamCraft2#357]
So, I'm doing some rethinking as 1.8.9 comes around. I wanted to highlight some areas that I want to change up, and try and get discussion going on how to change them, and perhaps on areas other people would like to see edited. I want to get a more cohesive feel for the mod going forward. To start off, I'm thinking about retheming the nuclear stuff, and redesigning or removing (or at least moving to BrassUtils) the engraving system. Thoughts?

**Comment by SkySom** [comment]
Nuclear to me just seems a bit strange to me. And if anything I'm not sure if the system is even that interesting? I mean isn't just put uranium in, get RF out? Though I'm not sure about the engraving, mainly cause I didn't even know it was a thing.

**Comment by FreneticScribbler** [comment]
Yeah, the system is vanilla as hell, and nuclear doesn't fit the theme in some ways.

Engraving was something that seems to have been copied by Surseance from the original steamcraft. The only reason I haven't cut it entirely is because I like some of Snurly's engraved textures :P

**Comment by SkySom** [comment]
I could see that as a valid reason to move them to Brass Utils.. Though that'll need a 1.8 move eventually (It's surprising larger than I was expecting). But yeah if we could make the nuclear more steampunkish while still keeping it an interesting system I'd say keep. Else wise, may be worth it to just scrap it.

**Comment by FreneticScribbler** [comment]
Yeah, I think I will move the engraving to Brass Utils. I'd say we try to rework the nuclear system before we scrap it.

**Comment by SkySom** [comment]
Start the ideas amigo! I'll do my best to help implement them. But I've got no ideas on how to improve the idea.

**Comment by FreneticScribbler** [comment]
That's the thing, for once I'm all out of ideas (for the nuclear stuff anyway) hence this issue. I'll keep thinking...

**Comment by Prospector** [comment]
Nuclear isn't really steampunk. Sorry I just don't see it.

**Comment by FreneticScribbler** [comment]
Yeah, it's much more atompunk. We still might be able to theme it so it's a bit closer to steampunk?

**Comment by mallrat208** [comment]
Random 3rd Party Comment:
If you keep nuclear, it may be best to just remove it as an actual power source but to keep it for some other applications.
- Uranium was at one point used in filaments for lights with that in mind you could use it in the production of lamps.
- Maybe special arrows or ammunition that either partially ignores armor or perhaps imparts a weakness effect on the victim.
- It's a rather dense material and relatively easy to machine so it wouldn't be out of place in a clockwork contraption as say a counterweight.

In terms of Power I think it'd be much more interesting, and unique in the terms of Minecraft mods, to expand upon Whaling for Oil. I may also have a soft spot for Dishonored however sooo.

**Comment by FreneticScribbler** [comment]
Third party feedback is just what I was after. 

Uranium is already used in AP rounds for the rocket launcher, lamp filaments and counterweights are interesting ideas though. 

Whaling will certainly be expanded in the future.

**Comment by HCSarise** [comment]
I agree about moving the engraving over.

More thoughts:

Flying Machines/Air Balloons etc: basically moving, pilotable airships and such. Now I realise this is already covered by mods like Archimedes ships and being able to make whatever design you like for that mod. However, the theme of this I think could still be covered without overlapping Archimedes too much. For example, having exosuit type functionality or jetpack type functionality in a pre-designed/determined air ship that you can craft upgrades and varying modules for. So you get a small moving base/airship but also with the upgrade/component functionality instead of just a pretty flying build. Alternatively, you could expand this theme into a speciality villager house (should you have need of a steamcraft2 villager), or make smaller flying machines be the theme for your "automata" or perhaps implement some kind of trade route/air ferry system of this style for servers (only cause I can't think of why single player would need a trade route/air ferry system).

Special dungeons/structure generation - something like a giant old dilapidated mech with dungeon rooms inside to fight through. You may also want to add in your own special steampunk themed mobs in these dungeons. Perhaps even make this one a separate mod/addon or at least config to turn it off. You could even scale this idea up to delapidated (or still functioning) flying cities if you wanted.

Trains - another version of the flying machines idea. We've got railcraft, sure, but that's just a bunch of minecarts linked together. I'm thinking more a giant steam engine kind of visual. Not sure how it's functionality would differ from what I said earlier though.

Mad Science - I'd like to see an expansion on the mad science side of things, more stuff along the lines of tesla coils and stuff that you've already got.

Body modding - I wish there was a way for a mod to affect what in-game Steve looks like by default. And I don't know if there's a way to provide upgrades/modules to the players themselves rather than having it be attached to an item. Change out one of your arms for a machine arm, machine eye, stuff like that. It's a cool idea I just don't know how possible it is.

**Comment by FreneticScribbler** [comment]
Pretty much all of those ideas I had vaguely considered, but you've got some nice detail ideas. That trade route idea is awesome! I'll be saving this for later, fits nicely with my long term plans. Gotta focus on polishing what we already got, for now, much as I dislike that.

**Comment by Prospector** [comment]
I'll be honest, I know very little about "steampunk" so I've been struggling a bit there, but I can certainly do some clockwork stuff. I feel like adding more clockwork stuff will make the mod a lot better. A big focus on the whole clockwork thing would be good.

**Comment by SkySom** [comment]
My question is why wouldn't uranium be more in place in say the eventual Dark New Age stuff

**Comment by FreneticScribbler** [comment]
Steampunk ain't just about clockwork :P But it's certainly a large part.

Uranium would definitely be more fitting in the far future stuff, but that's a long way out. Still somewhat hesitant to remove it.
